### PR TITLE
Java and python have been added to the Build complete samples yml file

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Review by Pawan-MSFT — APPROVED** [review]
looks correct, approving.

File: .github/workflows/build-complete-samples.yml [hunk 1]
```diff
@@ -849,3 +849,110 @@ jobs:
         run: |
           npm install
           npm run build --if-present
+
+  build-java-samples:
+    runs-on: windows-latest
+    #Add project path, name and version as below for any new additions
+    strategy:
+      matrix:
```
Comment (ChetanSharma-msft) [inline .github/workflows/build-complete-samples.yml:857]: @Harikrishnan-MSFT - If you have already run this pipeline, please add the screenshot of executed pipeline.

File: .github/workflows/build-complete-samples.yml [hunk 2]
```diff
@@ -849,3 +849,110 @@ jobs:
...
+        include:
+            - project_path: 'samples/bot-conversation/java'
+              name: 'bot-conversation'
```
Comment (ChetanSharma-msft) [inline .github/workflows/build-complete-samples.yml:860]: @Harikrishnan-MSFT - If you have already run this pipeline, please add the screenshot of executed pipeline.

File: .github/workflows/build-complete-samples.yml [hunk 3]
```diff
@@ -849,3 +849,110 @@ jobs:
...
+
+  build-python-samples:
+    runs-on: windows-latest
+    #Add project path, name and version as below for any new additions
+    strategy:
+      matrix:
+        include:
+            - project_path: 'samples/bot-conversation/python'
+              name: 'bot-conversation'
+              version: '3.10.x'
```
Comment (ChetanSharma-msft) [inline .github/workflows/build-complete-samples.yml:915]: Please add one empty line after every entry.

File: .github/workflows/build-complete-samples.yml [hunk 4]
```diff
@@ -849,3 +849,110 @@ jobs:
...
+            - project_path: 'samples/bot-message-reaction/java'
+              name: 'bot-message-reaction'
+              version: '18'
+            - project_path: 'samples/bot-teams-authentication/java'
+              name: 'bot-teams-authentication'
+              version: '18'
+            - project_path: 'samples/bot-file-upload/java'
+              name: 'bot-file-upload'
+              version: '18'   
+            - project_path: 'samples/bot-task-module/java'
```
Comment (ChetanSharma-msft) [inline .github/workflows/build-complete-samples.yml:871]: Please add one empty line after every entry.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my comments, please fix.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**